packages/core/aqua-compiler/src/utils.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ArrowWithoutCallbacks, NonArrowType, ProductType } from '@fluencelabs/interfaces';
+import { readFile } from 'fs/promises';
+import path from 'path';
+
+interface PackageJson {
+    version: string;
+    dependencies: {
+        ['@fluencelabs/aqua-api']: string
+    }
+}
+
+export async function getPackageJsonContent(): Promise<PackageJson> {
+    const content = await readFile(new URL(path.join('..', 'package.json'), import.meta.url));
+    const pkg = JSON.parse(content.toString());
+    return pkg;
+}
```

Comment:
This is a great example where JSON.parse returning any leads to shooting yourself in the foot like you did it here when you moved @fluencelabs/aqua-api to dev dependencies (for some reason tests don't fail, which must be fixed FIRST)

in my repo I use @total-typescript/ts-reset which changes JSON.parse so it returns unknown instead of any
this forces me to do the checks to make sure everything is working as expected

Also a small suggestion while we are already there
```suggestion
interface PackageJson {
    version: string;
    devDependencies: {
        ['@fluencelabs/aqua-api']: string
    }
}

export async function getPackageJsonContent(): Promise<PackageJson> {
    const content = await readFile(new URL(path.join('..', 'package.json'), import.meta.url), 'utf8');
    return JSON.parse(content);
}
```

Reply:
Yeah, really didnt know about utf-8 param, thanks :)